java/rocksjni.pom

```diff
@@ -4,26 +4,21 @@
         xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <modelVersion>4.0.0</modelVersion>
-    <name>RocksDB JNI</name>
-    <url>http://rocksdb.org/</url>
-    <groupId>org.rocksdb</groupId>
-    <artifactId>rocksdbjni</artifactId>
+    <name>FRocksDB JNI</name>
+    <url>https://github.com/dataArtisans/frocksdb</url>
+    <groupId>com.data-artisans</groupId>
+    <artifactId>frocksdbjni</artifactId>
     <!-- Version will be automatically replaced -->
     <version>-</version>
-    <description>RocksDB fat jar that contains .so files for linux32 and linux64, jnilib files
-        for Mac OSX, and a .dll for Windows x64.
+    <description>RocksDB fat jar to use with Apache Flink
+        that contains .so files for linux32 and linux64, jnilib files for Mac OSX, and a .dll for Windows x64.
     </description>
     <licenses>
         <license>
             <name>Apache License 2.0</name>
             <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
             <distribution>repo</distribution>
         </license>
-        <license>
```

Comment:
Is it valid to remove this? I am not an expert on licenses, so just a question :)

Reply:
Good question, me either, we do not deal with GNU at the moment, only Apache, so we decided to not include it.